ui/components/component-library/button-link/button-link.js

```diff
@@ -54,6 +57,10 @@ ButtonLink.propTypes = {
    * Default value is 'SIZES.AUTO'.
    */
   size: PropTypes.oneOf(Object.values(BUTTON_LINK_SIZES)),
+  /**
+   * textProps accepts all the props from Icon
```

Comment:
Update prop type description to align with prop
```suggestion
   * textProps accepts all the props from Text component
```